modules/gapi/misc/python/samples/gaze_estimation.py

```diff
@@ -0,0 +1,373 @@
+    heads_pos = processPoses.on(angles_y, angles_p, angles_r)
+
+    # Detect landmarks
+    landmark_inputs = cv.GInferInputs()
+    landmark_inputs.setInput('data', g_in)
+    landmark_outputs  = cv.gapi.infer('facial-landmarks', faces_rc, landmark_inputs)
+    landmark = landmark_outputs.at('align_fc3')
+
+    # Parse landmarks
+    left_eyes, right_eyes, mids, lmarks = parseEyes.on(landmark, faces_rc, sz)
+
+    # Detect eyes
+    eyes_inputs = cv.GInferInputs()
+    eyes_inputs.setInput('input.1', g_in)
+    eyesl_outputs  = cv.gapi.infer('open-closed-eye', left_eyes, eyes_inputs)
+    eyesr_outputs  = cv.gapi.infer('open-closed-eye', right_eyes, eyes_inputs)
+    eyesl = eyesl_outputs.at('19')
+    eyesr = eyesr_outputs.at('19')
+
+    # Process eyes states
+    l_eye_st, r_eye_st = getStates.on(eyesl, eyesr)
+
+    # Gaze estimation
+    gaze_inputs = cv.GInferListInputs()
+    gaze_inputs.setInput('left_eye_image'  , left_eyes)
+    gaze_inputs.setInput('right_eye_image' , right_eyes)
+    gaze_inputs.setInput('head_pose_angles', heads_pos)
+    gaze_outputs  = cv.gapi.infer2('gaze-estimation', g_in, gaze_inputs)
+    gaze_vectors  = gaze_outputs.at('gaze_vector')
+
+    out = copy.on(g_in)
+
+    comp = cv.GComputation(cv.GIn(g_in), cv.GOut(out,     \
+                                                 faces_rc, \
+                                                 left_eyes,  \
+                                                 right_eyes,  \
+                                                 gaze_vectors, \
+                                                 angles_y,      \
+                                                 angles_p,       \
+                                                 angles_r,        \
+                                                 l_eye_st,         \
+                                                 r_eye_st,          \
+                                                 mids, lmarks))
+
+    face_net      = cv.gapi.ie.params('face-detection'  , arguments.facem, weight_path(arguments.facem), arguments.faced)
+    head_pose_net = cv.gapi.ie.params('head-pose'       , arguments.headm, weight_path(arguments.headm), arguments.headd)
+    landmarks_net = cv.gapi.ie.params('facial-landmarks', arguments.landm, weight_path(arguments.landm), arguments.landd)
+    gaze_net      = cv.gapi.ie.params('gaze-estimation' , arguments.gazem, weight_path(arguments.gazem), arguments.gazed)
+    eye_net       = cv.gapi.ie.params('open-closed-eye' , arguments.eyem, weight_path(arguments.eyem), arguments.eyed)
+
+    kernels = cv.gapi.kernels(GparseEyesImpl, GprocessPosesImpl, GgetStatesImpl, GcopyImpl, Gscale_boxImpl)
+    nets = cv.gapi.networks(face_net, head_pose_net, landmarks_net, gaze_net, eye_net)
+
+    ccomp = comp.compileStreaming(args=cv.compile_args(kernels, nets))
+    source = cv.gapi.wip.make_capture_src(arguments.input)
+    ccomp.setSource(source)
+    ccomp.start()
+
+    cap = cv.VideoCapture(arguments.input)
```

Comment:
> make_capture_src
> VideoCapture

conflicting calls:
- should be reordered to probe properties before streaming
- VideoCapture should be closed first.

Reply:
`VideoCapture` is removed.